src/user/using-the-capture-app.md

```diff
@@ -878,6 +878,15 @@ You can reach the enrollment event form by typing in the address bar of your bro
 
 ![](resources/images/capture_app/enrollment-event-view-edit-url.png)
 
+### Error Widget
+
+On the enrollment dashboard, you can view the Error widget.
+
+![](resources/images/capture_app/enrollment-dash-error-widget-1.png)
+
+The error widget will display errors recieved from configured program rules which are not associated with a specific data element or attribute.
```

Comment:
I suggest simplifying and removing references to the DHIS2 data model:
`On the enrollment dashboard, the errors widget displays errors related to the current dashboard. The widget shows errors that are not associated with any specific data item.`